src/main.rs

```diff
@@ -36,8 +35,8 @@ async fn main() -> Result<(), Termination> {
     tracing::subscriber::set_global_default(subscriber)?;
 
     let opts = CliOpts::from_args();
-    tracing::trace!(opts = format_args!("{:#?}", opts));
```

Comment:
With getting rid of this `trace!`, we might as well git rid of making `opts` as well.

```
let CliOpts { dsts, addr } = CliOpts::from_args();
```